src/service/repo/signedMessageRepo.ts

```diff
@@ -44,8 +49,14 @@ async function checkIsTargetAllowingAnonDM(targetUserId: string) {
   }
 }
 
-async function sendSignedMessage(channelId: string, message: string, channelType: number) {
-  const payload = {channelId, message, channelType};
+async function sendSignedMessage(
+  channelId: string,
+  message: string,
+  channelType: number,
+  attachments: any
+) {
+  const payload = {channelId, message, channelType, attachments};
+  console.warn('payload', payload);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the implementation of `sendSignedMessage`, you are logging the entire payload which includes the message and attachments. This might expose sensitive user data in logs. Consider removing this log statement or only log necessary non-sensitive information.

```diff
-  console.warn('payload', payload);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->